support-frontend/assets/components/legal/termsPrivacy/termsPrivacy.jsx

```diff
@@ -8,16 +8,16 @@ import { privacyLink, contributionsTermsLinks, philanthropyContactEmail } from '
 import type { CountryGroupId } from 'helpers/internationalisation/countryGroup';
 import { type IsoCurrency, fromCountryGroupId, currencies } from 'helpers/internationalisation/currency';
 import type { ContributionType } from 'helpers/contributions';
-import { type CampaignName, campaigns } from 'helpers/campaigns';
 import './termsPrivacy.scss';
+import type { CampaignSettings } from 'helpers/campaigns';
 
 
 // ---- Types ----- //
 
 type PropTypes = {|
   countryGroupId: CountryGroupId,
   contributionType: ContributionType,
-  campaignName: ?CampaignName,
+  campaignSettings: ?CampaignSettings,
```

Comment:
As discussed, this would be better as `campaignSettings: CampaignSettings | null`

Reply:
👍